.github/workflows/mobile-curriculum-e2e.yml

```diff
@@ -5,8 +5,6 @@ on:
   pull_request:
     branches:
       - main
-    paths:
-      - "mobile-app/**"
```

Comment:
Could you undo this change? We don't want the e2e tests to run for any changes within the mobile API code

Reply:
Sorry for the delay, Niraj, but I've removed that commit.